firmware/tunerstudio/rusefi.input

```diff
@@ -987,23 +971,13 @@ gaugeCategory = Fuel Data
 		subMenu = std_separator
 		subMenu = std_realtime,    "&Realtime Display",              0
 		subMenu = tChargeSettings, "tCharge Settings"
+		subMenu = std_separator
+		subMenu = smLaunchControl,           "Launch Control"
 		
-	menu = "3D Map Tuning"
-;		subMenu = std_realtime,				"Realtime Display"
-;		subMenu = std_warmup,				"Warmup Wizard"
-		subMenu = fuelTableELMap,			"Fuel Table", 0, {fuelAlgorithm != 3}
```

Comment:
what is the plan for 3D map tuning? are we just removing it without moving anywhere?

Reply:
We're just removing the menu item, but 3D-view is still available as before:
![image](https://user-images.githubusercontent.com/15104768/37580458-6f269672-2b4c-11e8-977c-f85a3be4f081.png)
If a user likes to work with 3D-maps, there's no need to constantly check this option - it's saved as a preference.